### ISSUE TITLE
Dashboard command cannot ruin arbitrary SQL

### ISSUE THREAD

**Issue by whalito** [turbot/steampipe#2763]
Describe the bug
After updating Steampipe 0.16.1 to 0.17.1 with the described WSL update procedure, running the dashboard after upgrade, results in the following error message:
Error: dashboard command cannot run arbitrary SQL

**Steampipe version (`steampipe -v`)**
Example: v0.17.1

**To reproduce**
Run steampipe in v0.16.1 with all components working. Update to the latest version 0.17.1 and run steampipe dashboard start to see the error

**Expected behavior**
I expected to see steampipe working after the easy update procedure.

**Additional context**
No other context

**Comment by whalito** [comment]
Sure.

I did a plain and simple upgrade with the WSL bash command published on the update site.
And as advise from the CLI interface that there was update available towards 0.17.1.
Once successfully executed, the command steampipe dashboard start gave the mentioned ERROR sql arbitrary issue.
This only happens when executing the command ‘steampipe dashboard start or status’
Running steampipe in check or query mode, no issue has been reported.

As I use the dashboard service the most, I’d like to know how to fix it without reinstallation required.

Kr
Bert

**Comment by kaidaguerre** [comment]
@whalito I think I see the problem.

The command `steampipe dashboard start` is not in fact correct. To start the dashboard server in the foreground, you can just run `steampipe dashboard`.

To run in the background you can run `steampipe service start --dashboard`. To get the status of a the steampipe services you can run `steampipe service status`

In version 16 it would ignore the additional argument `start`. However in version 17 we added the ability to run individual dashboard by passing the dashboard name to the `dashboard` command, for example:

```
steampipe dashboard dashboard.aws_cloudtrail_trail_dashboard
```

What is happening here is that steampipe is looking for a resource in the mod matching the name `start` and as it fails to find one, it assumes you are trying to run a sql query directly (e.g. `steampipe dashboard "select * from aws_account"` ) which is not valid.

**Comment by kaidaguerre** [comment]
tl;dr try running `steampipe dashbnoard` or `steampipe service start --dashboard`

**Comment by whalito** [comment]
steampipe service start --dashboard does the trick
thanks for the swift support and answers
Closing the issue

**Comment by whalito** [comment]
solved by using steampipe service start --dashboard (twice) to enable the service again.
changes in cli were not listed in the update information
